validation/td-validation.ttl

```diff
@@ -93,7 +93,7 @@
                 is applied to TD instances."""^^rdf:HTML ;
         sh:datatype xsd:anyURI ;
         sh:maxCount 1 ;
-        sh:order 8 ;
+        sh:order 9 ;
```

Comment:
Note: This change is unrelated but I noticed afterwards follows yet another `sh:property` without `sh:order` and the one after that one starts with `sh:order 10` ;

It seems to be that we might need to add `sh:order` always and shift the numbers of the others? Am I wrong?